config/languages.dhall

```diff
@@ -335,24 +335,7 @@ in    [ { id = "assembly"
           , runCommand = "mcs -out:a.exe main.cs && mono a.exe"
           , versionCommand = "mcs --version"
           }
-        , books =
-          [ { purchaseLink = "https://www.amazon.com/Head-First-Learners-Real-World-Programming/dp/1491976705?_encoding=UTF8&psc=1&refRID=38VQ15A9W6GCP70CM4XZ&linkCode=li3&tag=glot06-20&linkId=4d5638a8b80d87c1bd000bac08553d41&language=en_US&ref_=as_li_ss_il"
-            , imgLink = "https://ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&ASIN=1491976705&Format=_SL250_&ID=AsinImage&MarketPlace=US&ServiceVersion=20070822&WS=1&tag=glot06-20&language=en_US"
-            , bookTitle = "Head First C#"
-            }
-          , { purchaseLink = "https://www.amazon.com/NET-Cross-Platform-Development-intelligent-Framework/dp/180056810X?_encoding=UTF8&psc=1&refRID=38VQ15A9W6GCP70CM4XZ&linkCode=li3&tag=glot06-20&linkId=a921bd41a6224bb4975a13d5b104ab98&language=en_US&ref_=as_li_ss_il"
-            , imgLink = "https://ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&ASIN=180056810X&Format=_SL250_&ID=AsinImage&MarketPlace=US&ServiceVersion=20070822&WS=1&tag=glot06-20&language=en_US"
-            , bookTitle = "C# 9 and .NET 5"
-            }
-          , { purchaseLink = "https://www.amazon.com/C-9-0-Nutshell-Definitive-Reference-dp-1098100964/dp/1098100964?_encoding=UTF8&me=&qid=&linkCode=li3&tag=glot06-20&linkId=d779db86705a99cd59d4260010dafbef&language=en_US&ref_=as_li_ss_il"
-            , imgLink = "https://ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&ASIN=1098100964&Format=_SL250_&ID=AsinImage&MarketPlace=US&ServiceVersion=20070822&WS=1&tag=glot06-20&language=en_US"
-            , bookTitle = "C# 9.0 in a Nutshell"
-            }
-          , { purchaseLink = "https://www.amazon.com/C-Depth-Jon-Skeet/dp/1617294535?_encoding=UTF8&psc=1&refRID=38VQ15A9W6GCP70CM4XZ&linkCode=li3&tag=glot06-20&linkId=36528b0a884047b94d78dc7f2657d589&language=en_US&ref_=as_li_ss_il"
-            , imgLink = "https://ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&ASIN=1617294535&Format=_SL250_&ID=AsinImage&MarketPlace=US&ServiceVersion=20070822&WS=1&tag=glot06-20&language=en_US"
-            , bookTitle = "C# in Depth"
-            }
-          ] : List Book
+        , books = [] : List Book
```

Comment:
It looks like you accidentally deleted the books from c#

Reply:
i have corrected changes